templates/base.html

```diff
@@ -1,91 +1,122 @@
+    {% for feed_filename in config.feed_filenames %}
+    <link type="application/atom+xml" rel="alternate" href="/{{feed_filename}}" title="{{config.title}}">
+    {% endfor %}
+    {% endif %}
     <!-- mathjax support -->
     <script>
         MathJax = {
-          tex: {
-            inlineMath: [['$', '$']]
-          }
+            tex: {
+                inlineMath: [['$', '$']]
+            }
         };
     </script>
     <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
-    </head>
-    <body cz-shortcut-listen="true">
-        <header class="site-header" role="banner">
-            <div class="wrapper">
-                {% block header %}
-                <a class="site-title" rel="author" href="/">{{config.title}}</a>
-                <nav class="site-nav">
-                    <input type="checkbox" id="nav-trigger" class="nav-trigger">
-                    <label for="nav-trigger">
-                        <span class="menu-icon">
-                            <svg viewBox="0 0 18 15" width="18px" height="15px">
-                                <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"></path>
-                            </svg>
-                        </span>
-                    </label>
-                    <div class="trigger">{% block navlinks %}
-                        <a class="page-link" href="/about">About</a>
-                        <a class="page-link" href="/blog">Blog</a>
-                        <a class="page-link" href="/wiki">Wiki</a>
-                    {% endblock navlinks %}</div>
-                </nav>
-                {% endblock header %}
-            </div>
-        </header>
-        <main class="page-content" aria-label="Content">
-            <div class="wrapper">
-                {% block content %}{% endblock content %}
-            </div>
-        </main>
-        <footer class="site-footer h-card">
-            <div class="wrapper">
-                {% block footer %}
-                <h2 class="footer-heading">{{config.title}}</h2>
-                <div class="footer-col-wrapper">
-                    <div class="footer-col footer-col-1">
-                        <ul class="contact-list">
-                            <li class="p-name">{{config.title}}</li>
-                            <li><a class="u-email" href="mailto:raph@levien.com">raph@levien.com</a></li>
-                        </ul>
-                    </div>
+    <!-- support dark mode and switching it in-app (switching not yet impl) -->
```

Comment:
If we're not going to support switching, then we should maybe do this out of javascript?

Does this block the initial load? (i.e. to make sure there isn't a wrong-styled flash)?